packages/core/src/local-admin-api.ts

```diff
@@ -7,38 +7,19 @@ import { LocalIndexApi } from './indexing/local-index-api.js'
  */
 export class LocalAdminApi implements AdminApi {
```

Comment:
this class is such a thin wrapper around the `LocalIndexApi` it's debatable whether it should even exist, but for the client right now we have a separation between the `client.index` methods and the `client.admin` methods, so need to mirror that separation server-side so long as they all share the same interfaces